modules/core/quick/farm_quick.install

```diff
@@ -0,0 +1,17 @@
+<?php
+
+/**
+ * @file
+ * Update functions for the farmOS Quick Form module.
+ */
+
+declare(strict_types=1);
```

Comment:
I like the idea of using `declare(strict_types=1)`, but since we do not use this anywhere else in farmOS I think we should omit it for now, just for consistency. But I would love to consider adopting it throughout farmOS and documenting what the benefits are in an issue...

Reply:
Removed the strict_types declaration.